miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/PaneAjaxTag.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2002-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.bbsrc.tgac.miso.webapp.util.form;
+
+import org.springframework.web.servlet.tags.RequestContextAwareTag;
+
+@SuppressWarnings("serial")
+public class PaneAjaxTag extends RequestContextAwareTag {
+
+  private String target;
+
+//  private String url;
+
+  @Override
+  protected int doStartTagInternal() throws Exception {
+
+//    pageContext.getOut().append(String.format(
+//        "<div id='%1$s'></div><script type='text/javascript'>jQuery(document).ready(function () { Pane.createInstrumentStatusPane('%1$s', PaneTarget.%2$s);});</script>",
+//        getId(), target));
+//    pageContext.getOut().append(String.format(
+//        "<div id='%1$s'></div><script type='text/javascript'>jQuery(document).ready(function () { Pane.createPane('%1$s', PaneTarget.%2$s);});</script>",
+//        getId(), target));
+    pageContext.getOut().append(String.format(
+        "<div id='%1$s'></div><script type='text/javascript'>jQuery(document).ready(function () { PaneTarget.%2$s.createPane('%1$s');});</script>",
+        getId(), target));
+    return SKIP_BODY;
+  }
+
+  public String getTarget() {
+    return target;
+  }
+
+//  public String getUrl() {
+//    return url;
+//  }
+
+  public void setTarget(String target) {
+    this.target = target;
+  }
+
+//  public void setUrl(String url) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)